### ISSUE TITLE
Printing weighting frame

### ISSUE THREAD

**Issue by MNstudent** [mbinkowski/nntimeseries#5]
Hello,

Your paper was an interesting read and thank you for making available the code. After following example.ipynb, I was wondering how I can print/visualize the weightings of the different 'candidate predictors'. It should be the case that B and D have high weightings for target variable E since E is a sum of last 10 values of B multiplied by D. 
Which variable in SOCNN.py is the weighting frame? And do you have any suggestions what the best way is to print this? 

Thank you in advance

**Comment by mbinkowski** [comment]
Hi, thanks for you interest in the paper.

To visualize the significance weights and offsets, you need to construct a keras function that recovers the proper part of the computational graph. significance weights are obtained as the output from 'softmax' layer, while offsets from 'offset1act' layer. The code may go as follows
```
significance_output = K.function([nn.layers[0].input, K.learning_phase()], [[l for l in nn.layers if l.name == 'softmax'][0].output])
offset_output = K.function([nn.layers[0].input, K.learning_phase()], [[l for l in nn.layers if l.name == 'offset1act'][0].output])
```
Where nn is the keras model of the network (stored by default in hdf5 files).
